### DISCUSSION TITLE
Releasing a new version: 0.1.8

### DISCUSSION THREAD

**Discussion by kaplanelad** [loco-rs/loco#199] [Announcements]
🎉 Loco version 0.1.7 was released! Explore the [changelog](https://github.com/loco-rs/loco/blob/master/CHANGELOG.md#v018) to discover exciting new features.

Highlights of this release include:

- Added`loco version` for getting an operable version string containing logical crate version and git SHA if available: 0.3.0 (<git sha>)
- Added: loco generate migration for adding ad-hoc migrations
- Added: added support in model generator for many-to-many link table generation via loco generate model --link
- Adding .devcontainer to starter projects

**Comment by norman784** [comment]
Thanks for the amazing work, it would be highly appreciated if he project uses GitHub releases, so it will be easier to be notified about the releases.
